subprojects/test-runner/client/src/main/kotlin/com/avito/runner/scheduler/metrics/TestRunnerMetricsListener.kt

```
@@ -34,24 +32,21 @@ internal class TestRunnerMetricsListener(
     }
 
     override suspend fun onDeviceCreated(device: Device, state: State) {
-        deviceWorkerEvents[device.key()] = DeviceWorkerEvent.Created(
-            created = timeProvider.nowInstant(),
-            testExecutionEvents = mutableMapOf(),
-        )
+        check(
```

Change logic to fail fast before make logging